### PR TITLE
FinalizingSmallDataset

### PR DESCRIPTION
**Pull request by andrewram4287**

### CODE REVIEW

File: gmm/figures/figure2.py [hunk 1]
```diff
@@ -1,17 +1,83 @@
...
+    components, vcexplained = runPCA(zflowDF)
+
+    ax[1].scatter(components, vcexplained)
+    xlabel = "Principal Components"
+    ylabel = "Variance"
+    ax[1].set(xlabel=xlabel, ylabel=ylabel)
+
+    # scoreDF(Determining rand_score and score for GMM with on DF and max cluster # with output [DF(Cluster #,Score)])
+    maxcluster = 18
+    scoreDF = cvGMM(zflowDF, maxcluster)
```
Comment (aarmey) [inline gmm/figures/figure2.py:49]: Why do you want to duplicate everything from Fig. 1?

Comment (andrewram4287) [reply gmm/figures/figure2.py:49]: Figure 1 has about 50 cells per experiment to see if analysis works, and Figure 2 is with a larger dataset of 6000 cell per experiment to see how graphs will change

Comment (aarmey) [reply gmm/figures/figure2.py:49]: Oh I see. That's fine to check out, but we should merge this once you've removed the redundancy and the CV is running with a smaller set. I'm guessing the CV plots won't change much, but it'll take ~1–2 hrs to run.

Comment (andrewram4287) [reply gmm/figures/figure2.py:49]: So keep CV for only smaller data set?

Comment (aarmey) [reply gmm/figures/figure2.py:49]: Yes. The "small" dataset is large as far as cvGMM is concerned, because it operates across all 300+ conditions.

**Pull request closed without merging**